app/javascript/mastodon/locales/ar.json

```diff
@@ -60,9 +60,12 @@
   "empty_column.public": "لا يوجد شيء هنا ! قم بتحرير شيء ما بشكل عام، أو اتبع مستخدمين آخرين في الخوادم المثيلة الأخرى لملء خيط المحادثات العام.",
   "follow_request.authorize": "ترخيص",
   "follow_request.reject": "رفض",
-  "getting_started.apps": "عدة تطبيقات مختلفة متوفرة",
```

Comment:
I am not sure you need a rebase, but looks like other locales got changed.
Also `manage:translations` is correct one, IMO.